src/routes/components/Modal.tsx

```diff
@@ -207,11 +243,13 @@ export default function TEMPLATE_Page() {
                   export class SomeOtherComponent {
                     open = false;
                     onClick(event: Event) {
-                      this.open = true;
+                      this.open = !this.open;
```

Comment:
This line is fine as is, since the button only ever opens the modal

Reply:
Done